scene/gui/text_edit.cpp

```diff
@@ -2812,6 +2812,17 @@ void TextEdit::clear() {
 }
 
 void TextEdit::_clear() {
+	if (editable && undo_enabled) {
+		_move_caret_document_start(false);
+		begin_complex_operation();
+
+		_remove_text(0, 0, MAX(0, get_line_count() - 1), MAX(get_line(MAX(get_line_count() - 1, 0)).size() - 1, 0));
+		insert_text_at_caret("");
+		text.clear(); 
```

Comment:
There's a trailing space here which should be removed to pass CI.

While doing it, it would be nice if you can amend the commit message to include a reference to TextEdit, so that it's clear when reading the changelog.

Reply:
Done. Hope it works now